src/PKSim.Core/Snapshots/Mappers/ExpressionContainerMapper.cs

```diff
@@ -55,18 +51,7 @@ private void mapTransporterExpressionProperties(ExpressionContainer snapshot, Tr
          if (transporterExpressionContainer == null)
             return;
 
-         snapshot.MembraneLocation = SnapshotValueFor(transporterExpressionContainer.MembraneLocation, _defaultMembraneLocation);
-      }
-
-      private bool shouldMapContainer(IParameter expressionParameter, TransporterExpressionContainer transportedExpressionContainer)
```

Comment:
This was not accurate. We should always export the membrane location . But RelativeExpression IsDefault means nothing to export